README.md

```diff
@@ -106,6 +106,13 @@ $ source $HOME/.cargo/env
 $ rustup component add rustfmt-preview
 ```
 
+If your rustc version is lower than 1.25.0, please update and install rustfmt:
+
+```bash
+$ rustup update
+$ rustup component add rustfmt-preview
```

Comment:
You won't need that line. It'll update on its own.

Reply:
@garious Okay, please fix on this, or update with new one.